README.md

```diff
@@ -214,11 +223,22 @@ spec:
         name: tpp-auth-secret
 ```
 
+**NOTE: The TPP URL can be found in the CloudShare > Environments > [ENVIRONMENT_NAME] > Windows Server 2016 > More Details > Web Access > Web SDK**. Copy and paste that URL into the Issuer file above.
+
 Now we can apply this configuration to the cluster using:
 ```console
 $ kubectl apply -f venafi-issuer.yaml
 ```
 
+Check that the Issuer has connected to TPP using:
+
+```console
+kubectl describe issuer venafi-tpp-issuer
+```
+
+You should see a message `Verified issuer with Venafi server`.
+If not, check for error messages and check that you have used the correct TPP credentials and URL.
```

Comment:
The user also might have to wait a few seconds, we should note that

Reply:
Fixed.